deploy/kube-config/influxdb/grafana.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         env:
         - name: INFLUXDB_HOST
           value: monitoring-influxdb
-        - name: GRAFANA_PORT
+        - name: GF_SERVER_HTTP_PORT
```

Comment:
This should be removed. Otherwise we will get an empty `GF_SERVER_HTTP_PORT` in `run.sh` cause `GRAFANA_PORT` is not set.